appveyor.yml

```diff
@@ -17,17 +17,16 @@ environment:
 
   matrix:
 
-      # We test Python 2.6 and 3.4 because 2.6 is most likely to have issues in
-      # Python 2 (if 2.6 passes, 2.7 virtually always passes) and Python 3.4 is
-      # the latest Python 3 release.
+      # We test Python 2.7 and 3.5 because 2.7 is the supported Python 2
+      # release and Python 3.5 is the latest Python 3 release.
```

Comment:
Maybe add a mention of Astropy in this sencente, to clear up why is 2.7 supposed to be the only supported version of python2 "...supported Python 2 release of Astropy and...".